java-samples/corda5-negotiation-cordapp/contracts/src/main/java/com/r3/developers/samples/negotiation/contracts/ProposalAndTradeContract.java

```diff
@@ -0,0 +1,100 @@
+package com.r3.developers.samples.negotiation.contracts;
+
+import com.r3.developers.samples.negotiation.states.Proposal;
+import com.r3.developers.samples.negotiation.states.Trade;
+import net.corda.v5.base.exceptions.CordaRuntimeException;
+import net.corda.v5.ledger.utxo.Command;
+import net.corda.v5.ledger.utxo.Contract;
+
```

Comment:
please reformat all the files you wrote, to remove uneeded newlines and make reading functions relatievely easier

Reply:
Applies to all files. In IntelliJ, `Code` -> `Reformat Code` is very useful. There's an option to format sections of code only